Dockerfile

```diff
@@ -0,0 +1,40 @@
+FROM jupyter/base-notebook:latest
```

Comment:
Why is the Dockerfile in the root of the repo?

Reply:
Binder needs the file to be in the root-directory in order to detect it (especially if other configuraiton-files like requirements.txt are present)